src/contact.js

```diff
@@ -0,0 +1,11 @@
+const contact = `   <div class="home-con">
+<h1>CONTACT</h1><br>
+<h5>Phone: 123 456 789</h5><br>
+
+<h5>Address: Adekule akala Ajao Estate 42, Lagos, NG</h5>
+<br><br>
+<h6>Open from:</h6><br>
+<h6>Monday till Friday</h6><br>
+<h6>From 08:00 till 20:00</h6>
+</div>`;
+module.exports = contact;
```

Comment:
Like in your other modules, in this module you created and exported a string instead of a function. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))